pages/doc/wavefront_release_notes.md

```diff
@@ -38,7 +38,10 @@ In October, 2023, we start to incrementally [**onboard**](csp_migration.html) al
 
 ## 2024-07.x Release Notes
 
-* **Ephemeral Internal Metrics**: All [internal metrics](wavefront-internal-metrics.html) are now [ephemeral](metric_types.html#metric-types-per-retention-period), so that they are retained for 28 days. Internal metrics are not convertible to persistent.
+**Ephemeral Internal Metrics**: Most of the [internal metrics](wavefront-internal-metrics.html) are now [ephemeral](metric_types.html#metric-types-per-retention-period) and not convertible to persistent. Exceptions are the following internal metrics, which remain persistent:
+```
```

Comment:
Ephemeral Internal Metrics**: Most of the [internal metrics](wavefront-internal-metrics.html) are now [ephemeral](metric_types.html#metric-types-per-retention-period) and can not converted to persistent except for the following internal metrics, which remain persistent: